src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -35,110 +37,37 @@ public static class OpenTelemetryServicesExtensions
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
         public static IServiceCollection AddOpenTelemetryTracing(this IServiceCollection services)
         {
-            services.AddOpenTelemetryTracing(builder => { });
-            return services;
-        }
-
-        /// <summary>
-        /// Adds OpenTelemetry TracerProvider to the specified <see cref="IServiceCollection" />.
-        /// </summary>
-        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
-        /// <param name="configure">The <see cref="TracerProviderBuilder"/> action to configure TracerProviderBuilder.</param>
-        /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
-        public static IServiceCollection AddOpenTelemetryTracing(this IServiceCollection services, Action<TracerProviderBuilder> configure)
-        {
-            if (configure is null)
+            services.AddSingleton<TracerProvider>((s) =>
```

Comment:
Stepping back question - for a console application it is possible to use multiple `TracerProvider` instances, with each provider having its own resource/instrumentation/sampler/processor.
Do we see a scenario here, or we always expect a singleton provider?

Reply:
If the Asp.Net Core (Or DI based ) requires more than one TracerProvider, they can still use the non DI method to create that. Don't think we need to support that using the DI extension methods. I don't see a need yet for that.